bot/bot.py

```diff
@@ -26,9 +28,7 @@ def __init__(self, **kwargs):
         )
 
     def load_extensions(self, exts: List[str]):
-        """
-        Unload all current cogs, then load in the ones passed into `cogs`
-        """
+        """Unload all current cogs, then load in the ones passed into `cogs`."""
```

Comment:
The function has "extensions" in the name so it's more consistent to use "extensions" over "cogs".

```suggestion
        """Unload all current extensions, then load the given extensions."""
```